open-company/open-company-web
# Attachments

___
### NS: Merge after #231 and #230 
___

Card: https://trello.com/c/T22m9Om2
Related PR: `open-company-api/attachments` TBD

To test:
- from dashboard edit a topic
- add a file from local device
- add another from google search or from gdoc
- check the list of attachments, everything looks correct? the attachments says `Draft`? good
- save the topic, do you see the creation date instead of `Draft`? good
- check `open-company-attachments` bucket of our S3, do you see the files you just uploaded? good
- refresh the page
- do you still see the list of files? good
- enter topic view for the topic, do you see the list of files? good
- edit the last entry
- remove a file
- add another
- refresh the page? do you see the expected files? good
- merge
- do one of Sean's funny things at the end of PRs

### NB: review with this https://github.com/belucid/open-company-infrastructure/pull/36
